documentation/SupportingDocuments/API_Documentation/User Stories/Edge Cloud APIs UserStory.md

```diff
@@ -0,0 +1,181 @@
+|Edge Application Management(EAM)|Edge Application Management API allows API consumers to manage the Life Cycle of an Application and to Discover Edge Cloud Zones. 
+|Simple Edge Discovery(SED)|The Simple Edge Discovery API returns the name of the closest operator Edge Cloud Zone to a particular user device.
+|Traffic Influence(TI)|The TI API provides the capability to establish the optimal routing, in terms of latency, in a specific geographical area, between the user Device, e.g. the user’s smartphone, and the optimal EAS instance nearby.
+
+To further information about each API, review the User story of each  one, which is available in the corresponding repository. 
+The purpose of this user story is to have a general picture of the life cycle management of the Edge cloud family APIs.
+
+## Actors, Roles, and Scope
+| Actor | Role | Scope |
+|-------|------|-------|
+|Application Provider (AP) |API Consumer | API Requester |
+|Edge Platform (EP)|Edge Cloud Provider | API Publisher |
+|Edge Service Provider (ESP)|Edge Cloud Provider|API Publisher
+|Operator|Communications Service Provider|API Publisher
+|User Client(UC)|API Consumer|API Requester
+
+Note: the role of Edge Service Provider may be played by either the MNO or Cloud Service Providers (Hyperscalers).
+
+
+#### Scenario 1: Request triggered by the AP and the Hyperscaler will host the App.
+
+
+| Item | Description | Support Qualifier |
+|----|----|----|
+|Summary|Edge Service Providers offers an Edge  platform that exposes computing and networking services in a given territory with several levels of abstraction being the Edge Cloud Zone the lowest one. Edge Service Providers offers a service through EdgeCloud APIs that permits application providers to deploy instances of an application in one or several Edge Cloud Zones, and to Applications Users the posibility to chose the most appropriate Edge Cloud Zone available at a given moment| M |
+|Pre-conditions|The AP has a resource usage agreement with the Operator| M |
+|Begins when|The user client request access to the application| M |
+|Step 1|The AP invokes Application Endpoint Discovery to request an Endpoint of the closest Edge Cloud Zone for this user(MSISDN)| M |
+|Step 2|The Application Endpoint Discovery returns to the EP an Endpoint with the closest Edge Cloud Zone for this user(MSISDN) | M|
+|Step 3|The AP instantiates the App in the Edge Cloud Zone determinated by the SED API| M|
+|Step 4|Once the App is istantiated the ESP returns the information about the Edge Cloud Zone where the App is instantiated| M|
+|Step 5|The AP provides the user client the corresponding URL to connected| M|
+|Step 6|The UE connects to the provided URL and start using the application| M|
+|Step 7|The AP invokes traffic influence API to activate the appropriate routing at the Edge of the mobile network toward the local ESP Instance for the Premium Users| M|

```

Comment:
As we are using both AP and user client we need to who can invoke which API. Here whether AP which may be an application too or the user client on UE may invoke the TI API. So just trying to understand if both AP and user client in mix are not creating slight confusion. Or at the beginning we should write who is AP and who is user client and what is there communication to be expected in this page.